exporter/datadogexporter/translate_traces.go

```diff
@@ -325,6 +326,17 @@ func resourceToDatadogServiceNameAndAttributeMap(
 		return true
 	})
 
+	// specification states that the resource level deployment.environment should be used for passing env,
+	// and also a number of Datadog UI components are hardcoded to point to /  look for / search with `env`.
+	// So we ensure that anytime deployment.environment is set, `env` is the same value.
+	// In the future, we can probably just set `env` using the deployment.environment value, but drop the duplicate
+	// deployment.environment tag afterward. Without knowing whether this would break existing user setups, its better
+	// to simply set both tags.
+	// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/deployment_environment.md#deployment
```

Comment:
Let's point to a given version of the spec here, since semantic conventions can change
```suggestion
	// https://github.com/open-telemetry/opentelemetry-specification/blob/v1.4.0/specification/resource/semantic_conventions/deployment_environment.md#deployment
```